src/test/javascript/spec/helpers/stubs/ngx-slider-stub.component.ts

```diff
@@ -0,0 +1,7 @@
+import { Component, Input } from '@angular/core';
+
+@Component({ selector: 'ngx-slider', template: '' })
```

Comment:
```suggestion
@Component({ selector: 'jhi-ngx-slider', template: '' })
```
Following our `jhi-ace-editor` component, which is a third party component as well

Reply:
but for the ace-editor this is consistent with the production code, for the slider it wouldn't be.

Reply:
Hmm, ok 👍